src/User32/User32+EnumDisplaySettingsExFlags.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+
+    /// <content>
+    /// Contains the <see cref="EnumDisplaySettingsExFlags"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// Flags for <see cref="EnumDisplaySettingsEx(string, uint, ref DEVMODE, EnumDisplaySettingsExFlags)"/>.
```

Comment:
nit: when specifying the overload, prefer the original one (the one with pointers).
Not a big deal nor something you have to correct on this PR, but something for next time.